File: comms/bulk-messaging/sender-pools/create-sender-pool/create-sender-pool.rb
require "twiliocomms"

client = TwilioComms::Client.new(
  account_id: "TWILIO_ACCOUNT_SID",
  auth_token: "TWILIO_AUTH_TOKEN"
)

client.sender_pools.create(
  name: "Sales Leads - APAC",
  tags: {
    region: "APAC"
  }
)

File: comms/bulk-messaging/sender-pools/create-sender-pool/create-sender-pool.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;
use Twilio\Comms\SenderPools\Requests\SenderPoolsCreateRequest;

$client = new TwilioComms(
    accountId: 'TWILIO_ACCOUNT_SID',
    authToken: 'TWILIO_AUTH_TOKEN',
);
$client->senderPools->create(
    new SenderPoolsCreateRequest([
        'name' => 'Sales Leads - APAC',
        'tags' => [
            'region' => 'APAC',
        ],
    ]),
);
